src/simulate/event_transformer.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+import json
+import logging
+from json import JSONDecodeError
+
+
+class Transformer:
+    def __init__(self, event):
+        self.event = event
+
+    def get_current_date_time_iso8601_format(self):
+        strftime = datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%S.%f')
```

Comment:
Rename your `strftime` variable to something else.  It's awkward for it to be named the same as the `strftime()` method.  Try `isoUtcNow` or `isoTimestamp` instead.